spring-cloud-generator/src/test/proto/compliance.proto

```diff
@@ -0,0 +1,232 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/cloud/extended_operations.proto";
+
+package google.showcase.v1beta1;
+
+option go_package = "github.com/googleapis/gapic-showcase/server/genproto";
+option java_package = "com.google.showcase.v1beta1";
+option java_multiple_files = true;
+
+// This service is used to test that GAPICs can transcode proto3 requests to
+// REST format correctly for various types of HTTP annotations.
+service Compliance {
```

Comment:
Why do we add a new proto? Is there any test cases can not be tested by `echo.proto`?

Reply:
No specific reason; I added the compliance proto because it was used in the gapic generator golden tests for REST-specific composers, but the spring composers don't currently implement anything that's specific to be tested by this proto. 

If the concern is to reduce the number of ptotos copied and maintained, these golden tests can switch to parse and use `echo.proto` (as `Transport.REST`).